modules/installation-aws-config-yaml.adoc

```diff
@@ -367,6 +368,9 @@ endif::vpc,restricted[]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
+ifndef::vpc,restricted[]
+<12> You can provide another load balancer type, such as `nlb`, in this parameter. The load balancer type is Classic by default.
```

Comment:
You can specify NLB or Classic. However default lbType is Classic if no value is provided.